test/track.test.js

```diff
@@ -214,6 +214,56 @@ describe('ravelin.track', function() {
         paste: 'hello',
         into: '<form action=/ name="form-name"><input type=password name=action></form>',
         props: {fieldName: 'action', formName: 'form-name', formAction: '/'}
+      },
+      {
+        paste: '4234 5678-9012 3456 7',
+        into: '<input name=custom-pan custom-pan-paste />',
+        props: {fieldName: 'custom-pan', selectionStart: 0, selectionEnd: 0, pastedValue: '0000 0000-0000 0000 0', panCleaned: true },
+        cfg: { 
+          classifyPaste: function(e) {
+            return {
+              pan: e.target.hasAttribute('custom-pan-paste')
+            };
+          }
+        }
+      },
+      {
+        paste: '4234 5678-9012 3456 7',
+        into: '<input name=custom-sensitive custom-sensitive-paste />',
+        props: {fieldName: 'custom-sensitive' },
+        cfg: { 
+          classifyPaste: function(e) {
+            return {
+              sensitive: e.target.hasAttribute('custom-sensitive-paste')
+            };
+          }
+        }
+      },
+      {
+        paste: 'hello',
+        into: '<input name=custom-sensitive custom-sensitive-paste />',
+        props: {fieldName: 'custom-sensitive' },
+        cfg: { 
+          classifyPaste: function(e) {
+            return {
+              pan: e.target.hasAttribute('custom-pan-paste'),
+              sensitive: e.target.hasAttribute('custom-sensitive-paste')
+            };
+          }
+        }
+      },
+      {
+        paste: 'hello',
+        into: '<input name=custom-pan custom-pan-paste />',
+        props: {fieldName: 'custom-pan', selectionStart: 0, selectionEnd: 0, pastedValue: 'XXXXX', panCleaned: false },
+        cfg: { 
+          classifyPaste: function(e) {
+            return {
+              pan: e.target.hasAttribute('custom-pan-paste'),
+              sensitive: e.target.hasAttribute('custom-sensitive-paste')
+            };
+          }
```

Comment:
I wonder if it would be clearer to have these test as `{pan: true, sensitive: true}` and so-on. Regardless, let's add a test case that returns `{pan: false, sensitive: false}`

Reply:
Yeah good point!